src/node_jsvmapi.cc

```diff
@@ -23,6 +23,24 @@
 #include <vector>
 #include <string.h>
 
+static v8::Persistent<v8::Value> *lastException = 0;
```

Comment:
It needs to be constructed with an `Isolate`, which is unavailable at the global scope.